src/ngraph/codegen/toolchains/MSVC.h

```diff
@@ -0,0 +1,161 @@
+//===--- MSVC.h - MSVC ToolChain Implementations ----------------*- C++ -*-===//
```

Comment:
Can we use the default header file for this one too?

Reply:
We can add NGraph header in this file and a comment as it did for some files in ngraph-tf: https://github.com/tensorflow/ngraph-bridge/blob/master/src/tf_deadness_analysis.cc Shall I do it?